test/summaryTest.js

```diff
@@ -63,6 +68,7 @@ describe('summary', function() {
 	});
 
 	it('should correctly use calculate other percentiles when configured', function() {
+		register.clear();
```

Comment:
never mind, I changed my mind - it's better to clear else there is a risk we will by mistake have the same name in another test.